src/pages/index.tsx

```diff
@@ -96,4 +104,32 @@ interface Props {
   img: string;
   featureText: string;
   featureHeader: string;
+  featureAltText: string;
 }
+
+interface HomepageProps {
+  data: HomepageData;
+}
+
+export const query = graphql`
+  query pageQuery {
+    page: markdownRemark(fields: { slug: { eq: "homepage" } }) {
+      id
```

Comment:
I don't see id or html being used anywhere. Can they be removed?

Reply:
Oops, you're right! I removed them and updated the PR.